test/types.documentarray.test.js

```diff
@@ -142,13 +142,6 @@ describe('types.documentarray', function() {
     assert.notEqual(a.id({ one: 'rolling', two: 'rock' }).title, 'rock-n-roll');
     assert.equal(a.id({ one: 'rock', two: 'roll' }).title, 'rock-n-roll');
 
-    // test with no _id
-    let NoId = new Schema({
```

Comment:
Need to remove `Noid` below too

Reply:
I had thought that was more so a test to see if it would work with the absence of an id since it was a variable assignment with _id = false and the comment says test with no _id. I'll remove it now.